Modules/System/Cryptography Management/src/DESCryptoServiceProvider.al

```diff
@@ -0,0 +1,68 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Provides helper functions for the Advanced Encryption Standard.
+/// </summary>
+
+codeunit 50102 "DESCryptoServiceProvider"
+{
+    Access = Public;
+
+    /// <summary>
+    /// Encrypts text with DESCryptoServiceProvider
+    /// </summary>
+    /// <param name="Password">Represents the password to be used to initialize a new instance of Rfc2898DeriveBytes</param>
+    /// <param name="Salt">Represents the salt to be used to initialize a new instance of Rfc2898DeriveBytes</param>
```

Comment:
The salt to use to initialize a new instance of the Rfc2898DeriveBytes <codeunit>.